esp/src/eclwatch/stub.js

```diff
@@ -87,6 +87,13 @@ define([
         }).play();
     }
 
+    function initUnlockListener() {
```

Comment:
Purpose of moving this?

Reply:
It was causing a lint error (called before declared).